packages/neon/neon_dashboard/lib/src/pages/main.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+import 'package:neon/blocs.dart';
+import 'package:neon/utils.dart';
+import 'package:neon/widgets.dart';
+import 'package:neon_dashboard/src/blocs/dashboard.dart';
+import 'package:neon_dashboard/src/widgets/widget.dart';
+
+class DashboardMainPage extends StatefulWidget {
+  const DashboardMainPage({
+    super.key,
+  });
+
+  @override
+  State<DashboardMainPage> createState() => _DashboardMainPageState();
+}
+
+class _DashboardMainPageState extends State<DashboardMainPage> {
+  late final DashboardBloc bloc;
+
+  @override
+  void initState() {
+    super.initState();
+
+    bloc = NeonProvider.of<DashboardBloc>(context);
+  }
+
+  @override
+  Widget build(final BuildContext context) => ResultBuilder.behaviorSubject(
+        stream: bloc.widgets,
+        builder: (final context, final widgets) => Center(
+          child: ListView(
```

Comment:
EDIT: looks like the `ListView` and `Wrap` are used to achieve a `GridView`.
Iirc `NeonListView.custom` should also accept `SliverGrid`s

Reply:
Well it should be a Wrap so we don't need to calculate how many items need to go in a row, but it would be possible to calculate this ourselves if you prefer that.

Reply:
There is a GridView fixed extend widget that forces a fixed width for the children and automatically calculates the fitting number of children.

You probably only thought of the fixed child count sliver builder.

Reply:
I think the following should help
https://api.flutter.dev/flutter/widgets/SliverGrid/SliverGrid.extent.html

Reply:
I'm not sure if I like it. The effect looks very ugly when resizing and everything is already on the same row. Also I don't really want it to change the size of the cards. They should just always have the same size.

Reply:
Please try out the current state. I'm not sure how to improve it so that is not this ugly to watch

Reply:
I can see; In this case flow is probably the right widget. Sorry for the confusion.
Nevertheless please use a `NeonListView` with a `SliverToBoxAdapter` like:

```dart
    return ResultBuilder.behaviorSubject(
      stream: bloc.widgets,
      builder: (final context, final snapshot) {
        Widget? child;
        if (snapshot.hasData) {
          child = Wrap(
            alignment: WrapAlignment.center,
            spacing: 8,
            runSpacing: 8,
            children: snapshot.requireData.entries
                .map(
                  (final widget) => DashboardWidget(
                    widget: widget.key,
                    items: widget.value,
                  ),
                )
                .toList(),
          );
        }

        return Center(
          child: NeonListView.custom(
            isLoading: snapshot.isLoading,
            error: snapshot.error,
            onRefresh: bloc.refresh,
            sliver: SliverToBoxAdapter(
              child: child,
            ),
          ),
        );
      },
    );
```